tests/sanity/test/explorer/test-api-gateway.js

```diff
@@ -10,18 +10,17 @@
 
```

Comment:
It's great to test gateway/version endpoint. Can we move this to `tests/sanity/test/apiml` folder?

Reply:
Also can we rename `tests/sanity/test/explorer` to `tests/sanity/test/explorer-apis`. Now `Explorer` is also name of Zowe vscode extension.